### DISCUSSION TITLE
Akses Mecha-cms.com

### DISCUSSION THREAD

**Discussion by ghost** [mecha-cms/mecha#133] [Q&A]
Halo, Mas Taufik, situs mecha-cms.com kok tidak bisa diakses ya? Saya coba DTE juga. Apa yang bermasalah IP saya ya?

**Comment by taufik-nurrohman** [comment]
Dari sini bisa pak. Mungkin sedang _down_. Maklum, _shared hosting_.

**Reply by ghost** [reply]
Saya kira IP saya yang masalah. Soalnya saya kerapkali mengunjungi situs Mecha. Tapi saya belum coba akses pakai VPN untuk mengetahui apakah IP saya yang terlalu sering mengunjungi Mecha dan DTE. 😊
